Libraries/Share/Share.js

```diff
@@ -58,8 +58,8 @@ class Share {
    */
   static share(content: Content, options: Options = {}): Promise<Object> {
     invariant(
-      typeof content === 'object' && content !== null,
```

Comment:
Why this change? What would be the case for content being of type 'object' but not strictly being equal to 'null'?